src/api/l_math_mat4.c

```diff
@@ -155,22 +155,18 @@ static int l_lovrMat4Mul(lua_State* L) {
     return 4;
   } else {
     float v[4];
-    int index = luax_readvec3(L, 2, v, "number, vector, or Mat4");
+    int index = luax_readvec3(L, 2, v, "number, vector");
```

Comment:
This should still say `or Mat4`, so that if you do `m:mul("hi")`, you get something like:

```
Bad argument #1 to mul (number, vector, or Mat4 expected, got string)
```